src/main/java/org/primefaces/extensions/component/timeago/TimeAgo.java

```diff
@@ -0,0 +1,185 @@
+/**
+ * Copyright 2011-2019 PrimeFaces Extensions
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.primefaces.extensions.component.timeago;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Locale;
+import javax.faces.application.ResourceDependencies;
+import javax.faces.application.ResourceDependency;
+import javax.faces.component.UIComponentBase;
+import javax.faces.context.FacesContext;
+import org.primefaces.component.api.Widget;
+import org.primefaces.extensions.util.MessageFactory;
+import org.primefaces.util.ComponentUtils;
+import org.primefaces.util.ResourceUtils;
+
+/**
+ * <code>TimeAgo</code> component.
+ *
+ * @author Jasper de Vries &lt;jepsar@gmail.com&gt;
+ * @since 7.0.1
+ */
+@ResourceDependencies({
+    @ResourceDependency(library = "primefaces", name = "jquery/jquery.js"),
+    @ResourceDependency(library = "primefaces", name = "core.js"),
+    @ResourceDependency(library = "primefaces-extensions", name = "timeago/jquery.timeago.js"),
```

Comment:
@melloware Can you compress these two JavaScripts? I'm still don't fully get how that plugin works.

Reply:
Yep I will take care of it